### DISCUSSION TITLE
Add support for BiaxialLineChart

### DISCUSSION THREAD

**Discussion by stefanosambruna** [buildo/bento-design-system#479] [Ideas]
We need a chart with two different vertical axis scales, one for every line.

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/24191264/208126879-f7167b7c-3d12-4a42-8957-5b6d5390415a.png">

![image](https://user-images.githubusercontent.com/24191264/208127248-a2d3b0c7-242b-475b-9880-719d82370f91.png)

**Comment by gabro** [comment]
So, one possible workaround for this is to add `recharts` as a dependency in your app, and pass a custom `YAxis` as children of bento's `LineChart`. Can you try that and see if it works as expected?

**Reply by stefanosambruna** [reply]
mmmh I think it won't work cause you have to specify the relations between axes and lines with `yAxisId`, am I wrong?

**Reply by gabro** [reply]
I'm not sure actually, but each `Line` has a unique `dataKey` so if you specify the same `dataKey` for the axis it might work?
